table/builder.go

```diff
@@ -109,6 +111,16 @@ func (b Builder) keyDiff(newKey []byte) []byte {
 }
 
 func (b *Builder) addHelper(key []byte, v y.ValueStruct) {
```

Comment:
Is this function getting reused?

Reply:
It is used in `Add` and `finishBlock` function in `builder.go` file